Userland/Libraries/LibWeb/Selection/Selection.cpp

```diff
@@ -152,6 +152,9 @@ void Selection::add_range(JS::NonnullGCPtr<DOM::Range> range)
 
     // 3. Set this's range to range by a strong reference (not by making a copy).
     set_range(range);
+
+    // AD-HOC: WPT selection/removeAllRanges.html and selection/addRange.htm expect this
```

Comment:
Can you/did you post a spec bug about this?

Reply:
I don't know how to